packages/devtools_app/lib/src/screens/memory/primitives/painting.dart

```diff
@@ -4,30 +4,23 @@
 
 import 'package:flutter/material.dart';
 
-const _totalDashWidth = 15.0;
+// TODO(polina-c): use the same constants as for dash in the chart.
 const _dashHeight = 2.0;
-const _dashWidth = 4.0;
+const _dashWidth = 5.0;
 const _spaceBetweenDash = 3.0;
 
 Widget createDashWidget(Color color) {
-  return Container(
```

Comment:
Do you know why we had a `painter` and `foregroundPainter`? Can you try removing one of these painters and see if you have the same issue?

Reply:
I tried this and many other things. It is bug in Flutter, they could repro it (see https://github.com/flutter/flutter/issues/116494).

Reply:
Okay, just wanted to make sure!